usermanager/src/main/java/org/pesmypetcare/usermanager/clients/pet/PetManagerClient.java

```diff
@@ -224,13 +227,14 @@ public int deleteFieldCollection(String accessToken, String username, String pet
         PetData.checkCollectionField(field);
         taskManager = taskManager.resetTaskManager();
         taskManager.setTaskId(DELETE);
-        StringBuilder response = taskManager.execute(BASE_URL + PETS_PATH + username + SLASH
-                + petName + "/collection/" + field, accessToken).get();
+        StringBuilder response = taskManager
+                .execute(BASE_URL + PETS_PATH + username + SLASH + petName + "/collection/" + field, accessToken).get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/collection/" appears 7 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5561031)